ci/axis/gpu.yml

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2018-2020 NVIDIA Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Released under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+
+SDK_TYPE:
+  - nvcr.io/nvidia/cuda
```

Should just be `cuda`
```suggestion
  - cuda
```

---

Fixed.